django/cantusdb_project/main_app/views/chant.py

```diff
@@ -348,6 +348,7 @@ def get_context_data(self, **kwargs):
                         </td>
                     </tr>
                 """
+                print(gregorien_response)
```

Comment:
You probably want to remove this print statement.

Reply:
Thanks for catching this! Done!